### ISSUE TITLE
Lock file maintenance PR not opened for a while

### ISSUE THREAD

**Issue by edmorley** [renovateapp/renovate#2033]
**What Renovate type are you using?**
Renovate GitHub App

**Describe the bug**
We use `:maintainLockFilesWeekly`, however the both today and last Monday there was been no new PR.

**To Reproduce**

1.  Watch https://github.com/mozilla-neutrino/neutrino-dev for new lockfile maintenance PRs.

**Expected behavior**
A new PR every Monday.

**Actual behavior**
Last PR was mozilla-neutrino/neutrino-dev#850, meaning there have been two missed since then. 

Though strangely our other repositories have had their PRs:
mozilla/treeherder#3583
mozilla-neutrino/webpack-chain#60.

**Comment by rarkins** [comment]
Hi @edmorley, thanks for catching this. I refactored `lerna` support a couple of weeks ago and this was caught up in it. In short: I made the decision to *always* call npm/yarn via `lerna bootstrap`, instead of selectively deciding whether to call it or npm/yarn directly. However the lock file maintenance logic never included lerna so repos like this are for now missing out. I'll try to add the necessary logic soon.

Closing as duplicate of #1974
